src/fides/api/ops/task/graph_task.py

```diff
@@ -726,12 +726,14 @@ def termination_fn(*dependent_values: int) -> Tuple[int, ...]:
                 t.erasure_request,
                 access_request_data[
                     str(k)
-                ],  # Pass in the results of the access request for this collection
-                *[
-                    access_request_data[
-                        str(upstream_key)
-                    ]  # Additionally pass in the original input data we used for the access request. It's helpful in
+                ]  # Pass in the results of the access request for this collection
+                if str(k) in access_request_data
+                else [],  # handle case with no read entry, i.e. disabled connector
```

Comment:
Good catch avoiding this key error, could be written as this instead, a get with a default if it isn't found.
```
access_request_data.get(str(upstream_key), [])
```

Reply:
ah yeah, that's a lot cleaner! soon enough `.get()` will become instinct for me :)